apps/frontend/app/(client)/(main)/contest/[contestId]/@tabs/leaderboard/_components/LeaderboardRow.tsx

```diff
@@ -0,0 +1,90 @@
+'use client'
+
+import BronzeMedal from '@/public/icons/medal-bronze.svg'
+import GoldMedal from '@/public/icons/medal-gold.svg'
+import SilverMedal from '@/public/icons/medal-silver.svg'
```

Comment:
icon src 가져올 때 컨벤션 준수해주세용
camelCase이고 Icon으로 끝나야 합니다
```suggestion
import bronzeMedalIcon from '@/public/icons/medal-bronze.svg'
import goldMedalIcon from '@/public/icons/medal-gold.svg'
import silverMedalIcon from '@/public/icons/medal-silver.svg'
```

Reply:
좋아요! 금방 수정할게요!